examples/program_generator/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "demo_program_generator"
+version = "0.1.0"
+authors = ["Gear Technologies"]
+edition = "2018"
+license = "GPL-3.0"
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies]
+galloc = { path = "../../galloc", default-features = false }
+gcore = { path = "../../gcore", default-features = false }
+gstd = { path = "../../gstd", features = ["debug"] }
+codec = { package = "parity-scale-codec", version = "3.1.2", default-features = false, features = ["derive"] }
+hex-literal = "*"
```

Comment:
Not too good practice to hope that a major release will not break backward compatibility.

Reply:
done